scripts/deploy-on-kind.sh

```diff
@@ -189,7 +189,7 @@ function deploy_korifi() {
     helm upgrade --install korifi helm/korifi \
       --namespace korifi \
       --values="$values_file" \
-      --set=adminUserName="cf-admin" \
```

Comment:
The reason was to simplify local setup. We can just use pre-created user by kind, called `kubernetes-admin`

Reply:
Is this strictly needed for the purpose of splitting korifi controllers to multiple deployments. I am not sure if all korifi dev scripts will work with `kubernetes-admin`. Maybe we can do this simplification as a separate PR so that we make sure we don't break anything.

Reply:
sounds good